cdist/conf/type/__pacman_integrate/manifest

```diff
@@ -0,0 +1,46 @@
+state=$(cat $__object/parameter/state 2>/dev/null || echo "present" )
+
+if [ "${state}" == "present" ]; then
+    #__rsync /etc/pacman.conf\
+    __file /home/rouxdo/Documents/Work/cdist-dev/pacman.conf\
```

Comment:
the paths look rather relative to your home directory :)

Maybe use "$__type/files/..."